zxing-android-embedded/AndroidManifest.xml

```diff
@@ -14,28 +13,43 @@
  See the License for the specific language governing permissions and
  limitations under the License.
  -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.google.zxing.client.android">
-
-  <uses-permission android:name="android.permission.CAMERA"/>
-
-  <!-- Don't require camera, as this requires a rear camera. This allows it to work on the Nexus 7 -->
-  <uses-feature android:name="android.hardware.camera" android:required="false"/>
-  <uses-feature android:name="android.hardware.camera.front" android:required="false"/>
-  <!-- TODO replace above two with next line after Android 4.2 -->
-  <!-- <uses-feature android:name="android.hardware.camera.any"/> -->
-  <uses-feature android:name="android.hardware.camera.autofocus" android:required="false"/>
-  <uses-feature android:name="android.hardware.camera.flash" android:required="false"/>
-  <uses-feature android:name="android.hardware.screen.landscape" android:required="false" />
-  <uses-feature android:name="android.hardware.wifi" android:required="false"/>
-
-
-  <application>
-      <activity android:name="com.journeyapps.barcodescanner.CaptureActivity"
-                android:clearTaskOnLaunch="true"
-                android:screenOrientation="sensorLandscape"
-                android:stateNotNeeded="true"
-                android:theme="@style/zxing_CaptureTheme"
-                android:windowSoftInputMode="stateAlwaysHidden"/>
-  </application>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.google.zxing.client.android">
+
+    <uses-permission android:name="android.permission.CAMERA" />
+
+    <!-- Don't require camera, as this requires a rear camera. This allows it to work on the Nexus 7 -->
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.camera.front"
+        android:required="false" />
+    <!-- TODO replace above two with next line after Android 4.2 -->
+    <!-- <uses-feature android:name="android.hardware.camera.any"/> -->
+    <uses-feature
+        android:name="android.hardware.camera.autofocus"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.camera.flash"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.screen.landscape"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.wifi"
+        android:required="false" />
+
+
+    <application>
+        <activity
+            android:name="com.journeyapps.barcodescanner.CaptureActivity"
+            android:clearTaskOnLaunch="true"
+            android:exported="true"
```

Comment:
This one should not be exported - it's only intended to be used from inside the same app.